integration-tests/bats/sql-merge.bats

```diff
@@ -71,25 +71,104 @@ SELECT DOLT_COMMIT('-a', '-m', 'this is a ff');
 SELECT DOLT_CHECKOUT('main');
 SELECT DOLT_MERGE('feature-branch');
 SELECT COUNT(*) > 0 FROM test WHERE pk=3;
+
+SELECT COUNT(*) from dolt_status;
```

Comment:
Move this into a `dolt sql -q`  below the other test. Then assert there's nothing in dolt_status

Reply:
You can also do something clever with something like 
```
test-nickname $ dolt sql -r csv -q "select hash from dolt_branches where branch='main'"
hash
4mgglt0kr56l9c0joff7dt2nbjt815bh
```
do another select from dolt_branches for feature-branch , and then assert that the two hashes are the same.